apps/mobile/locales/en.json

```diff
@@ -336,6 +337,7 @@
   "This project is being managed on": "This project is being managed on",
   "This will just take a moment": "This will just take a moment",
   "To use the support chat": "To use the support chat,",
+  "To use the support chat, please enable the support cookie in your preferences": "",
```

Comment:
This translation got nuked? And these are mobile i18n files?

Reply:
odd